NEWS

```diff
@@ -7,6 +7,12 @@
 # history (i.e. packaged versions of the software).
 #
 
+usbutils 013
```

Comment:
No need to change this file at all, I create this when I do a new release.

Reply:
I guessed it, but don't want to left the old number for my local installed version.

Reply:
I removed my changes ...